js/src/hooks/useExistingGoogleAdsAccounts.js

```diff
@@ -15,8 +15,11 @@ const useExistingGoogleAdsAccounts = () => {
 		const isResolving = select( STORE_KEY ).isResolving(
 			'getExistingGoogleAdsAccounts'
 		);
+		const hasFinishedResolution = select( STORE_KEY ).hasFinishedResolution(
```

Comment:
We can do that and I have tested this by modifying the hook, but the issue is that this hook is being used by few other components as well and when we use `useAppSelectDispatch` it changes the the return data (property names etc.), so if we accommodate that change in other component, this will be out of the scope of the ticket especially considering the scenarios where we may also have to fix few tests.

So we can handle this in a separate issue.